packages/CB-serverless-frontend/src/components/categories/index.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+import Checkbox from 'material-ui/Checkbox';
+import CircularProgress from 'material-ui/CircularProgress';
+
+import ProductItem from '../ProductItem';
+import SubCategories from './sub-categories';
+
+const ItemsWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: center;
+  align-content: center;
+  padding-bottom: 1em;
+  margin: 1em auto;
+  box-shadow: 0 0 26px 0 #eee;
+  background: #eee;
+  width: 85%;
+`;
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding-bottom: 1em;
+  margin: 1em auto;
+  box-shadow: 0 0 26px 0 #eee;
+  background: #eee;
+  height: 100%;
+`;
+
+class Categories extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: [],
+      subCategories: [],
+      checked: {},
+      fetchingData: true
+    };
+  }
+
+
+  componentDidMount() {
+    const {category} = this.props;
+    let catData = {};
+    axios.get('http://localhost:3000/groceries?category=' + category)
```

Comment:
We need to move data fetching into sagas, since at last everything will be graphQL end points, plus use environment based API's and move to config file